packages/integration-tests/src/entities/types.ts

```diff
@@ -7,6 +8,10 @@ export const address = object({
   street: string(),
 });
 
+export const AddressSchema = z.object({
+  street: z.string(),
```

Comment:
Doesn't have to be for this PR, but might be nice to have a schema/integration test that explicitly cover a schema with different input & output types.